pik-shop-backend/src/main/java/pl/elka/pw/pik/shop/controllers/HelloWorldController.java

```diff
@@ -1,14 +1,14 @@
-package pl.elka.pw.pik.shop;
+package pl.elka.pw.pik.shop.controllers;
```

Comment:
Nie to do wyrzucenia ale poki co zostawilem
Moze jeszcze Igor bd chcial obczaic HelloWorlda